MdeModulePkg/Include/Library/CapsuleLib.h

```diff
@@ -50,6 +50,27 @@ ProcessCapsuleImage (
   IN EFI_CAPSULE_HEADER  *CapsuleHeader
   );
 
+// MU_CHANGE - Add a routine to allow firmware-specific handling for capsules
+//           that need to persist across reset.
+
+/**
+  The firmware-specific implementation that stages the capsule image
+  for processing after reset if it recognized the format of this capsule
+  image.
+
+  Caution: This function may receive untrusted input.
+
+  @param  CapsuleHeader    Pointer to the UEFI capsule image to be processed.

```

Comment:
Could you please add the direction to the parameters in this commit?

``suggestion
  @param[in]  CapsuleHeader    Pointer to the UEFI capsule image to be processed.
```